Dockerfile

```diff
@@ -1,18 +1,18 @@
 # Build the binaries in larger image
-FROM docker.io/fortio/fortio.build:v13 as build
+FROM hub.docker.prod.walmart.com/fortio/fortio.build:v13 as build
```

Comment:
please don't make those changes

I am curious though, which change did you make to the build image ?

Reply:
There's no change. But in walmart we only have internal docker proxy. I built a docker image for my change.
Sorry for not cleaning up this.